Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16 AS builder
+WORKDIR /go/src/github.com/forbole/big-dipper
```

I think we can change the `WORKDIR` to be something related to BDJuno instead. Maybe its GitHub repository path: 

```suggestion
WORKDIR /go/src/github.com/forbole/bdjuno
```